assets/company-admin/components/CompanyAdminApp.tsx

```diff
@@ -32,8 +31,38 @@ import {UserListSortFilter} from 'users/components/filters/UserListSortFilter';
 import EditUser from 'users/components/EditUser';
 import {CompanyDetailsProductFilter} from './CompanyDetailsProductFilter';
 import {CompanyAdminProductSeatRequestModal} from './CompanyAdminProductSeatRequestModal';
+import {IUser} from 'interfaces/user';
 
-class CompanyAdminAppComponent extends React.Component<any, any> {
+interface IProps {
+    sectionId: any;
+    setSection: any;
+    editUser: any;
+    newUser: IUser;
+    setError: any;
+    saveUser: any;
+    deleteUser: any;
+    resetPassword: any;
+    resendUserInvite: any;
+    closeUserEditor: any;
+    usersById: any;
+    user: IUser;
+    userToEdit: IUser;
+    errors: any;
+    products: any;
+    modal: any;
+    setQuery: any;
+    fetchUsers: any;
+    sort: any;
+    sortDirection: any;
+    setSort: any;
+    toggleSortDirection: any;
+    productId: any;
+    setProductFilter: any;
+    companies: any;
+    totalUsers: any;
+}
+
+class CompanyAdminAppComponent extends React.Component<IProps, any> {
```

Comment:
state only has a few fields, if you're touching the component already it'd be nice to type state too